docs/content/docs/recipes/teams.md

```diff
@@ -176,25 +176,25 @@ export const <your tab name> = () => {
 }
 ```
 
-1. Replace `TODO 1` with the following code. For the content page to display in Teams, we must include the [Microsoft Teams JavaScript client SDK](https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true) and include a call to initialize it after your page loads.
+1. Replace `TODO 1` with the following code. For the content page to display in Teams, you must include the [Microsoft Teams JavaScript client SDK](https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true) and include a call to initialize it after your page loads.
```

Comment:
for all the docs.microsoft.com URLs, you should remove the en-us from the URL so that people get localized sites when appropriate.